### PR TITLE
formatting + new descriptions

### PR DESCRIPTION
**Pull request by untuned**

### CODE REVIEW

File: commands/a.js [hunk 1]
```diff
@@ -2,24 +2,24 @@ const Discord = require('discord.js')
...
-if (!message.member.roles.find(r => r.name.toLowerCase() === 'staff')) {
-    message.channel.send(new Discord.RichEmbed().setColor("RANDOM").setDescription('You are missing permission.'))
-    return
-}
-if (!message.channel.topic.includes("Commission #") && !message.channel.topic.includes("Ticket #")) {
-    message.channel.send(new Discord.RichEmbed().setColor("RANDOM").setDescription('Invalid channel.'))
-    return
-}
 
-const previousname = message.channel.name
```
Comment (MegaJoshy) [inline commands/a.js]: Issue that shouldn't be removed

Comment (MegaJoshy) [reply commands/a.js]: Maybe not

Comment (untuned) [reply commands/a.js]: it was moved down a line

File: commands/a.js [hunk 2]
```diff
@@ -2,24 +2,24 @@ const Discord = require('discord.js')
...
-  message.channel.send(new Discord.RichEmbed().setColor("RANDOM").setDescription('Thank you for using Ather Network, if you need anymore help just reply to this message. We will get back to you shortly. Otherwise just close this ticket with `-close`.'))
-} else {
-  message.channel.setName(`completed-${previousname}`)
-  message.channel.send(new Discord.RichEmbed().setColor("RANDOM").setDescription('Thank you for using Ather Network, if you need anymore help just reply to this message. We will get back to you shortly. Otherwise just close this commission with `-close`.'))
-}
+    const previousname = message.channel.name
 
+    if (!message.channel.topic.includes("Commission #")) {
+        message.channel.setName(`ans-${previousname}`)
+        message.channel.send(new Discord.RichEmbed().setColor("RANDOM").setDescription('Commission marked as answered.  If you have any other questions, please reply to this channel.  When finished, you may close it with `-close`.'))
```
Comment (MegaJoshy) [inline commands/a.js]: This is for ticket not commission

Comment (MegaJoshy) [inline commands/a.js]: This is for ticket not commission

File: commands/a.js [hunk 3]
```diff
@@ -2,24 +2,24 @@ const Discord = require('discord.js')
...
+    } else {
+        message.channel.setName(`completed-${previousname}`)
+        message.channel.send(new Discord.RichEmbed().setColor("RANDOM").setDescription('Ticket marked as answered.  If you have any other questions, please reply to this channel.  When finished, you may close it with `-close`.'))
```
Comment (MegaJoshy) [inline commands/a.js]: Commission not ticket.

File: commands/a.js (hunk 1 above)
Comment (MegaJoshy) [reply commands/a.js]: ik

**Pull request merged**